server/middleware/query.go

```diff
@@ -0,0 +1,35 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/medibloc/panacea-oracle/panacea"
+	log "github.com/sirupsen/logrus"
+)
+
+type queryMiddleware struct {
+	panaceaQueryClient panacea.QueryClient
+}
```

Comment:
Please name this more clearly, such as `queryHeightMiddleware` or so.

Reply:
OK. I fixed it(https://github.com/medibloc/panacea-oracle/pull/52/commits/6fff09740b975cc9ad94a8bd67e3490aab77458b)